### PR TITLE
Change british grey to american gray

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

**Review by adellibovi — APPROVED** [review]
Yaaay 🍾

File: latest/files.json [hunk 1]
```diff
@@ -1 +1 @@
...
....png","Apple iPhone 6s Space Gray.png","Apple iPhone 6s Plus Gold.png","Apple iPhone 6s Plus Rose Gold.png","Apple iPhone 6s Plus Silver.png","Apple iPhone 6s Plus Space Gray.png","Apple iPhone 7 Gold.png","Apple iPhone 7 Jet Black.png","Apple iPhone 7 Matte Black.png","Apple iPhone 7 Rose Gold.png","Apple iPhone 7 Silver.png","Apple iPhone 7 Plus Gold.png","Apple iPhone 7 Plus Jet Black.png","Apple iPhone 7 Plus Matte Black.png","Apple iPhone 7 Plus Rose Gold.png","Apple iPhone 7 Plus Silver.png","Apple iPhone 8 Gold.png","Apple iPhone 8 Silver.png","Apple iPhone 8 Space Gray.png","Apple iPhone 8 Plus Gold.png","Apple iPhone 8 Plus Silver.png","Apple iPhone 8 Plus Space Gray.png","Apple iPhone SE Gold.png","Apple iPhone SE Rose Gold.png","Apple iPhone SE Silver.png","Apple iPhone SE Space Gray.png","Apple iPhone X Silver.png","Apple iPhone X Space Gray.png","Apple-Macbook-Gold.png","Apple-Macbook-Space-Gray.png","Apple iPad Air 2 Gold.png","Apple iPad Air 2 Silver.png","Apple iPad Air 2 Space Gray.png","Apple iPad Mini 4 Gold.png","Apple iPad Mini 4 Silver.png","Apple iPad Mini 4 Space Gray.png","Apple iPad Pro Gold.png","Apple iPad Pro Silver.png","Apple iPad Pro Space Gray.png"]
```
Comment (mpirri) [inline latest/files.json]: What's the difference here?

Comment (ohayon) [inline latest/files.json:1]: Instances of "Grey" are now "Gray"

**Review by mpirri — APPROVED** [review]
👍

**Pull request merged**